Detections/MultipleDataSources/PhishiglinkExecutionObserved.yaml

```diff
@@ -0,0 +1,95 @@
+id: 2fed0668-6d43-4c78-87e6-510f96f12145
+name: Phishing link click observed in None Microsoft Network device
+description: |
+  'The purpose of this content is to identify successful phishing links accessed by users. Once a user clicks on a phishing link, we observe successful network activity originating from non-Microsoft network devices. These devices may include Palo Alto Networks, Fortinet, Check Point, and Zscaler devices.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId:  OfficeATP
+    dataTypes:
+    - SecurityAlert
+  - connectorId:  PaloAltoNetworks
+    dataTypes:
+    - CommonSecurityLog (PaloAlto)
+  - connectorId:  Fortinet
+    dataTypes:
+    - CommonSecurityLog (Fortinet)
+  - connectorId:  CheckPoint
+    dataTypes:
+    - CommonSecurityLog (CheckPoint)
+  - connectorId:  Zscaler
+    dataTypes:
+    - CommonSecurityLog (Zscaler)
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+relevantTechniques:
+  - T1071
+query: |
+    //Finding security alerts and extracting the Entities 
+    SecurityAlert
+    //extracting Alert Entities 
```

Comment:
Given the size of this table would an initial filter to check entities has the items we want be more efficent?

Reply:
Also to meet our specified scenario we want to filter this to phishing related alerts as you have done in the other query. I know that other query has them as well but we need one that just does the alert to network connection as we defined in our scoping doc.